monitors/c/src/internals/process_binary.h

```diff
@@ -0,0 +1,13 @@
+#ifndef R2U2_ENGINES_PROCESS_BINARY_H
+#define R2U2_ENGINES_PROCESS_BINARY_H
+
+#include "errors.h"
+#include "../memory/monitor.h"
+
+/// @brief      Reads from spec binary, filling out instruction memory and instruction table
+/// @param      spec Pointer to a vector or array of uintt_8 from the specification compiled by C2PO
+/// @param[in]  monitor  Pointer to r2u2_monitor_t
```

Comment:
```suggestion
/// @param[out]  monitor  Pointer to r2u2_monitor_t
```

Data is written to the monitor and the output is not contingent on the state of monitor, only of spec